aou/aou-rstudio-bioconductor/rstudio/scripts/default_user.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -e
+
+DEFAULT_USER=${1:-${DEFAULT_USER:-"rstudio"}}
```

Comment:
what is default user?  
The RStudio we use will disable auth: 
https://github.com/DataBiosphere/terra-app/blob/main/apps/rstudio/app.yaml#L10L16
The file above is the RStudio config Leo will install for us when we call Leo.

Reply:
If you want to dig into the details of how the rocker dockers are setup it's better you ask them
https://github.com/rocker-org/rocker-versioned2/blob/23b6961dcc187b7290b35ae4434180dfb1fa7f24/scripts/default_user.sh

All the script files are copied here because we had to change the base image.

Reply:
And we do exactly the same
```
ENV DISABLE_AUTH=true
ENV ROOT=false
```

Reply:
The contents of this PR is identical to `us.gcr.io/broad-dsp-gcr-public/anvil-rstudio-bioconductor:3.15.3` with the exception that we have a different base image with wondershaper (hence why we have to copy-paste their scripts) and we add 25 more packages in the end.
That's it.

Reply:
default user is `rstudio` and is run as a non-root user with defaults `1000:1000` in `S6`